src/main/java/com/gorlah/kappabot/subcommand/SkeletalKappaBotCommand.java

```diff
@@ -0,0 +1,130 @@
+package com.gorlah.kappabot.subcommand;
+
+
+import com.gorlah.kappabot.command.Command;
+import com.gorlah.kappabot.command.CommandPayload;
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Objects;
+import java.util.TreeMap;
+
+public abstract class SkeletalKappaBotCommand implements Command {
```

Comment:
I think I would just call this one "AbstractCommand"

Reply:
Good idea.